movies/src/resources/doc.jsonld

```diff
@@ -6,10 +6,52 @@
   "hydra:description": "This API shows how a basic CRUD (Create, Read, Update, Delete) API can be designed using Hydra Core Vocabulary.",
   "hydra:entrypoint": "/",
   "hydra:supportedClass": [
+    {
+      "@id": "schema:EntryPoint",
+      "@type": "hydra:Class",
+      "hydra:title": "API entrypoint",
+      "hydra:description": "Every capability of the API can be reached from here by following links.",
+      "hydra:supportedProperty": [
+        {
+          "hydra:property": "hydra:collection",
+          "hydra:title": "Collections",
+          "hydra:description": "Collections provided by this API"
+        }
+      ]
+    },
+    {
+      "@id": "mov:MovieCollection",
```

Comment:
Unless this was your intention, this won't tell any client it is a collection. Neither 'hydra:Collection` nor any of the predicates with that type in domain is provided. I'd prefer explicit declarstion, as it makes clieants easier to implement.

Reply:
Indeed I provided the additional `@type`. `MovieCollection` is only used to add some operations it does not have to be a `hydra:Collection` itself.

Reply:
That's what I've wrote. If it was ypur intention, no client will recognize this resource as a collection. But a human tends to treat it alike. I' change the @id of thos resource to something less collection related

Reply:
How will it not treat it as collection if the resource is **explicitly** typed as one?

Reply:
Hmm. Maybe I'm missing somethig. All I can see is in the API documentation is that resource `mov:MovieCollection` is of type `hydra:Class` and no other trace of a collection is available on that level. Indeed, `index.jsonld` file mentions, that resource `movies` is of types `hydra:Collection` and `mov:MovieCollection`, but this information is not available from the API documentation. This is what I wrote - if that was an intention - let us change `mov:MovieCollection` to something that does not mention collection directly. It distracts (well, I was fooled at first to think of it as a collection!)

Reply:
This is but just a name. It does not change anything if you rename it to `mov:Movies` or `mov:MovieContainer` (LDP terminology).

Besides, it is exactly that - an extension to the `hydra:Collection` class. We do not support nor need subclassing hence they are detached from one another.

> no other trace of a collection is available on that level

Why do you need it explicit in the API Doc? The client always looks at the compete picture: doc + resource representation.

Reply:
I aggree that the naming might confuse. I think I like the `mov:Movies` proposal, as a human reader might indeed think of LDP when choosing to call it `mov:MovieContainer`

Reply:
Like I said, IMO it's intended for machines to process and humans to understand and thus doesn't really matter all that much.

That being consistent, I should agree to whatever change you guys decide to make 😁

Reply:
Renamed it to `mov:Movies` and adjusted the description